### PR TITLE
Dave/condition telemetry request

### PR DESCRIPTION
**Pull request by davetsay**
## Overview
Addresses #2642
- Provides historical data as condition set output from LAD (if available) of each telemetry criterion

## Author Checklist
| | |
| --- | :---: |
| Changes address original issue? | Y |
| Unit tests included and/or updated with changes? | Y |
| Command line build passes? | Y |
| Changes have been smoke-tested? | Y |

### CODE REVIEW

File: src/plugins/condition/ConditionSetTelemetryProvider.js [hunk 1]
```diff
@@ -9,21 +9,33 @@ export default class ConditionSetTelemetryProvider {
...
+                return latestOutput ? [latestOutput] : [];
+            });
+    }
+
     subscribe(domainObject, callback) {
         let conditionManager = new ConditionManager(domainObject, this.openmct);
         conditionManager.on('conditionSetResultUpdated', callback);
 
-        return function unsubscribe() {
+        return () => { // unsubscribe function
```
Comment (shefalijoshi) [inline src/plugins/condition/ConditionSetTelemetryProvider.js]: I kind of like how you had this before - calling this `unsubscribe` by name. Makes it clearer.

File: src/plugins/condition/ConditionManager.js [hunk 2]
```diff
@@ -31,23 +31,32 @@ export default class ConditionManager extends EventEmitter {
...
     initialize() {
-        this.conditionResults = {};
-        this.observeForChanges(this.domainObject);
-        this.conditionCollection = [];
-        if (this.domainObject.configuration.conditionCollection.length) {
-            this.domainObject.configuration.conditionCollection.forEach((conditionConfigurationId, index) => {
-                this.openmct.objects.get(conditionConfigurationId).then((conditionConfiguration) => {
-                    this.initCondition(conditionConfiguration, index)
-                });
+        this.openmct.objects.get(this.domainObject.identifier)
```
[the remark this answers is not in the document]
Comment (davetsay) [reply src/plugins/condition/ConditionManager.js]: done. removed the call

File: src/plugins/condition/ConditionSetTelemetryProvider.js (hunk 1 above)
Comment (davetsay) [reply src/plugins/condition/ConditionSetTelemetryProvider.js]: sure. changed back.

**Comment by davetsay** [comment]
This is now stale. Re-opening as #2759

**Pull request closed without merging**